django/template/base.py

```diff
@@ -395,7 +395,6 @@ def tokenize(self):
                 token_string = self.template_string[upto:start]
                 result.append(self.create_token(token_string, (upto, start), lineno, in_tag=False))
                 lineno += token_string.count('\n')
-                upto = start
```

Comment:
`upto` variable is override at https://github.com/django/django/pull/11320/files#diff-8cbe1fb6d589cb6e35b956704d7a1285L402 in all cases so no need of this variable.